### DISCUSSION TITLE
Different results from same circuit in latest version

### DISCUSSION THREAD

**Discussion by CoolBassist** [hneemann/Digital#1040] [Q&A]
Hello,
I seem to get different results from my circuit that is just simply reading from a rom, decoding the instructions, and then executing them. However the values at the registers seem to be not consistent. Most of the time it works fine, but in 1 out of 10 times, it doesn't. Is this a known issue?

Kind regards.

**Comment by hneemann** [comment]
I suspect you sometimes read data from a high impedance wire/bus. This actually leads to random values and is intended that way.

You would have to attach the circuit to the issue to say more about this.
